docs/scipp-neutron/overview.rst

```diff
@@ -4,7 +4,7 @@
 Overview: scipp.neutron
 =======================
 
-``scipp-neutron`` builds on top of ``scipp-core`` and provides features specific to handling data generated in neutron scattering facilities.
+``scipp-neutron`` builds on top of the core ``scipp`` package and provides features specific to handling data generated in neutron scattering facilities.
```

Comment:
This will be accurate in future. Note at the moment there is no separate package for scipp core. Do we want to just say "scipp core functionality"?

Reply:
I think removing the reference to `scipp-core` is good, since we do not have such a Python package. Apart from that I don't think this matters much, since we are going to change it anyway.

Reply:
But will it be called `scipp-core`? Internally, this library doesn't even know about Variables...

Reply:
No, it will just be `scipp`, that is we I say I think it is good to remove the mention of `scipp-core`. The difference will be that there will be no `neutron` submodule, it would be `scipp_neutron` or something like that.